src/main/scala/it/pagopa/interop/attributeregistryprocess/service/impl/AgreementManagementServiceImpl.scala

```diff
@@ -0,0 +1,40 @@
+package it.pagopa.interop.attributeregistryprocess.service.impl
```

Comment:
[same](https://github.com/pagopa/interop-be-attribute-registry-process/pull/1/files#r1135229414)